operator/sp1/sp1.go

```diff
@@ -29,9 +29,16 @@ func VerifySp1Proof(proofBuffer []byte, elfBuffer []byte) (isVerified bool, err
 	}()
 
 	proofPtr := (*C.uchar)(unsafe.Pointer(&proofBuffer[0]))
+	//publicInputsPtr := (*C.uchar)(unsafe.Pointer(&publicInputsBuffer[0]))
```

Comment:
This commented code could be deleted if it's not needed

```suggestion
	//publicInputsPtr := (*C.uchar)(unsafe.Pointer(&publicInputsBuffer[0]))
```